src/models/eventReportPilot.js

```diff
@@ -0,0 +1,42 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class EventReportPilot extends Model {
+    static associate(models) {
+      EventReportPilot.hasMany(models.TrainingReportPilot, { foreignKey: 'eventId', as: 'trainingReports' });
+    }
+  }
+
+  EventReportPilot.init({
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    ownerId: {
```

Comment:
yeah, this relationship will get an overhaul with the generic collaborator changes, so it might be easier to temporarily forego creating this linkage and instead handle creating the relationship when we migrate later on to the generic solution